docs/UserGuide.md

```diff
@@ -143,18 +188,33 @@ Format: `s-list`
 
 Finds all drinks where their abbreviated names match the KEYWORD(s).
 
-Format: `s-find KEYWORD [MORE_KEYWORDS] ...`
+Format: `s-find KEYWORD [MORE_KEYWORDS]`
 
-* The search is case-insensitive. e.g `bsbbt` will match `BSBBT`.
-* Only the drink's abbreviated name is searched.
-* Only full words will be matched e.g. `BSB` will not match `BSBBT`.
-* Drinks matching at least one keyword will be returned.
-  e.g. `BSBBT BSBM` will return `BSBBT`, `BSBM`.
+**:information_source: Note:** <br>
+The search is case-insensitive. e.g `bsbbt` will match `BSBBT`.<br>
+Only the drink's abbreviated name is searched.<br>
+Only full words will be matched e.g. `BSB` will not match `BSBBT`. <br>
+Drinks matching at least one keyword will be returned.
+  e.g. `BSBBT BSBM` will return `BSBBT`, `BSBM`.<br>
```

Comment:
No need to be placed in a note box? Perhaps need to standardise how the *-find features are explained, @WM71811  did not put them in a note box.

I also suggest placing them in bullet points instead.